TACC-Cloud/hazmapper
# Feat/tup 700 core components node pkg  use pkg  fixes

## Overview: ##

This PR makes a few changes to https://github.com/TACC-Cloud/hazmapper/pull/264.  It works along side https://github.com/TACC/tup-ui/pull/484

## PR Status: ##


## Related Jira tickets: ##

* [TUP-700](https://tacc-main.atlassian.net/browse/TUP-700)

## Summary of Changes: ##

## Testing Steps: ##
1. `rm -rf node_modules`and then `npm install`
2. use the dev (`GeoapiBackendEnvironment.Dev`) or staging backend (`GeoapiBackendEnvironment.Staging`) in react/src/secret_local.ts
3. `npm run dev`
4.  confirm you can login, see map listings and view a map.

## Notes: ##

- [x] publish 0.0.3 version of tacc/core-components
- [x] use 0.0.3 here and test and ensure github actions are working
- [x]  fix build ` [cause]: Error: Cannot find module '@rollup/rollup-linux-x64-gnu' `
